run

```diff
@@ -14,7 +14,7 @@ let args = process.argv.slice(2)
 let skip_validation = '--skip-version-validation'
 async function init () {
     let skipValidationIndex = args.indexOf(skip_validation)
-    if (skip_validation === -1) {
+    if (skipValidationIndex === -1) {
```

Comment:
This is the output I get with this PR:
```
enso$ node --version
v14.17.6
enso$ git log | head -n5
commit 618764d35da8ac75eac4b9105ee7b57f65fa039b
Author: Jaroslav Tulach <jaroslav.tulach@enso.org>
Date:   Fri Apr 8 10:49:18 2022 +0200

    Mention rustup, wasm-pack and cargo-watch installation command
enso$ ./run  start --dev



--------------------------
--- Version Validation ---
--------------------------

Use the `--skip-version-validation` flag to skip it.
Querying npm for the latest LTS versions.
Checking versions of installed packages.
Checking if 'node' version is 'v16.14.2'.

unknown:0
unknown
^
undefined
(Use `node --trace-uncaught ...` to show where the exception was thrown)
```
e.g. without the option `--skip-version-validation` the execution really fails.